benches/appender.rs

```diff
@@ -58,7 +58,13 @@ fn mean(numbers: &[u64]) -> f32 {
 fn appender_mean_eager(backend: BackendType, window_size: usize, b: &mut Bencher) {
     let dir = tempdir().unwrap();
     with_backend_type!(backend, |B| {
-        let backend = Arc::new(B::create(dir.as_ref()).unwrap());
+        let backend = Arc::new(
+            B::create(
+                dir.as_ref(),
+                Box::leak(backend.to_string().into_boxed_str()),
```

Comment:
```suggestion
                "appender_backend"
```

This is just benchmarks, so the name of the backend can just be anything. For example, name it the same as the type of benchmark.